frontend/TestRun/ResultsTab.svelte

```diff
@@ -43,18 +100,15 @@
         }
     };
 
-    const getColumnType = (result, columnName) => {
-        return result.meta.columns_meta.find((col) => col.name === columnName).type;
-    };
-
     onMount(() => {
         fetchResults();
     });
 
     const styleMap = {
```

Comment:
Const values ideally should be uppercase split by underscore: `STYLE_MAP`. Also you can move this into `Common/TestStatus.js` and import it here, that's where a lot of those helper objects are.

Reply:
fixed